gazelle/python/testdata/binary_without_entrypoint_per_file_generation_partial_update/a.py

```diff
@@ -0,0 +1,3 @@
+if __name__ == "__main__":
+    print("Hello, world!")
+
```

Comment:
This extra newline is not significant, correct? If not please remove it (lest I come back and look at this later and think that it is, haha).

Reply:
Ahh nope, not significant!